lib/publiccloud/azure.pm

```diff
@@ -24,6 +24,12 @@ has storage_account => 'openqa';
 has container       => 'sle-images';
 has lease_id        => undef;
 
+sub decode_json_trim {
+    my $json = shift;
+    $json =~ s/.{4}$//;
```

Comment:
btw, this is a bug in the cli it seems. I reported a bug 
https://github.com/Azure/azure-cli/issues/9903

maybe you can put it as a comment, to explain why we need this function.

Reply:
Also, I would be careful with this and not trim the last 4 characters always. What if this behavior stops happening? we might end up trimming characters we don't want. Maybe you can add a condition `if " [0m" == last 4 chars, then trim`.

Reply:
```suggestion
    $json =~ s/.{4}$// if ($json =~ /. \[0m$/);
```

Reply:
ok